api/api.go

```diff
@@ -259,45 +263,12 @@ type Orb struct {
 	Source string `json:"-"`
 	// Avoid "Version" since there is a "version" key in the orb source referring
```

Comment:
Should we delete this comment? I realize you didn't change this but it doesn't seem relevant now?